### ISSUE TITLE
Atom 'Script' package issue

### ISSUE THREAD

**Issue by menon95** [atom/atom#17365]
Hi,

I'm pretty new to python programming and github etc and am not really familiar with this environment.

I have an issue with my Atom text editor when running Python. Specifically i use python 3.5 and have Atom to write my code. I have installed a package called 'Script' to run the python scripts in the editor itself.

I attempted to run a simple script but it gives me an error as follows:

![capture](https://user-images.githubusercontent.com/39441498/40271142-4fe532d2-5b99-11e8-8f4d-bc809b6048ee.JPG)

I tried a bunch of stuff but i'm not really savvy with it yet so please do give me some advice! Better yet if you could dumb it down a little bit.

Many thanks!!

**Comment by 50Wliu** [comment]
This question is more suited for [Discuss](https://discuss.atom.io), the official Atom and Electron message board.  I would recommend that you create a new topic there.

That said, try following the answers in [this StackOverflow question](https://stackoverflow.com/questions/5599872/python-windows-importerror-no-module-named-site).
